pkg/bot/interactive/help.go

```diff
@@ -47,9 +47,9 @@ func Help(platform config.CommPlatformIntegration, clusterName, botName string)
 					Description: fmt.Sprintf("You can run kubectl commands directly from %s!", cases.Title(language.English).String(string(platform))),
```

Comment:
BTW The default configuration has `kubectl` disabled. Should we hide the section in such case? I think we should, but it will be easy to fix I think 🤔 I see at least two options:

1. Explicitly return to the user that kubectl functionality is disabled for this cluster. Don't show the commands, or have the buttons disabled.
2. At least add another sentence to the description that the kubectl should be enabled (if the first option would be too difficult).

First option is much better and should be easy to implement. Current behavior can be confusing - if you click the button, you'll get `Command not supported. Please use 'help' to see supported commands.` 😄

Reply:
Let's address that in folllow-up PR and I will sync with @lpetkov about the approach for that.

1. "(..)  have the buttons disabled." - that's not possible, however we are able to don't show the commands and "(..) should be easy to implement." - yep, but requires a bit of new code and e2e testing.

Reply:
A follow-up PR totally works for me 👍I just wanted to make sure we won't forget about this 👍